source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,74 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Grouping elastic profile by plugin-id', 3772%>
+
+  This provides an organized view of the elastic profiles making it easier to find a particular elastic agent.
+
+  <figure class='shrink-figure-70'>
+      <%= image_tag 'releases/17.9.0/elastic-agent-grouping-by-plugin-id.png', alt: 'Group elastic profile by plugin-id' %>
+  </figure>
+
+* <%= link_to_commit 'ca7c3fa232087b67553e7d30bbfb409ba5aab49d', 'Addition of a request processor to allow plugins to query the server info', 3401%>
+
+* <%= link_to_commit '1c1f2fe0d4ebda46ddfe0766ce50d580039a3f77', 'Admins can go to pipelines edit page directly from VSM'%>
+
+  <figure class='shrink-figure-70'>
+      <%= image_tag 'releases/17.9.0/edit-pipeline-from-vsm.png', alt: 'Edit pipeline from VSM' %>
+  </figure>
+
+* <%= link_to_commit 'd47180f2193239ddd318157c43d3f60b6168d9e0', 'Addition of a few more fields to materials form which is used in creation of pipelines.'%>
+
+<h4>API Changes</h4>
+* <%= link_to_commit '0587f21042701d9ee1e64ae698341c495df6c8d8', 'Plugin info api v3', 3719%>
```

Comment:
Enhanced the [Plugin Info API] (link to commit) to include extension related information.